src/scribe_data/language_data_extraction/Kurmanji/verbs/query_verbs.sparql

```diff
@@ -5,9 +5,78 @@
 SELECT
   (REPLACE(STR(?lexeme), "http://www.wikidata.org/entity/", "") AS ?lexemeID)
   ?verb
+  ?form
+  ?grammaticalFeature
+  ?infinitive
+  ?directCase
+  ?gerund
+  ?intransitivePhase
+  ?basicPhase
+  ?conjParticiple
+  ?adverbial
+  ?absConstruction
+  ?accusative
+  ?ergative
+  ?additivePhase
 
 WHERE {
   ?lexeme dct:language wd:Q36163 ;
     wikibase:lexicalCategory wd:Q24905 ;
     wikibase:lemma ?verb .
+
+  
+     # Infinitive (basic form of the verb)
+  FILTER(lang(?verb) = "ku") .
+
+    # Forms associated with the grammatical features (direct case, gerund, etc.)
+  OPTIONAL {
+    ?lexeme ontolex:lexicalForm ?formNode .
+    ?formNode ontolex:representation ?form ;
+             wikibase:grammaticalFeature ?grammaticalFeature .
+    FILTER(LANG(?form) = "ku") .
+  }
+  
+  OPTIONAL {
+    ?lexeme ontolex:lexicalForm ?adverbialForm .
+    ?adverbialForm ontolex:representation ?adverbial ;
+    wikibase:grammaticalFeature wd:Q380012 .
+    FILTER(LANG(?adverbial) = "ku") .
+  } .
+
+  # MARK: Absolute Construction
+
+  OPTIONAL {
+    ?lexeme ontolex:lexicalForm ?absConstructionForm .
+    ?absConstructionForm ontolex:representation ?absConstruction ;
+    wikibase:grammaticalFeature wd:Q4669807 .
+    FILTER(LANG(?absConstruction) = "ku") .
+  } .
+
+  # MARK: Accusative
+
+  OPTIONAL {
+    ?lexeme ontolex:lexicalForm ?accusativeForm .
+    ?accusativeForm ontolex:representation ?accusative ;
+    wikibase:grammaticalFeature wd:Q1233197 .
+    FILTER(LANG(?accusative) = "ku") .
+  } .
+
+  # MARK: Ergative
+
+  OPTIONAL {
+    ?lexeme ontolex:lexicalForm ?ergativeForm .
+    ?ergativeForm ontolex:representation ?ergative ;
+    wikibase:grammaticalFeature wd:Q1233197 .
+    FILTER(LANG(?ergative) = "ku") .
+  } .
+  
+  
+  
+  
+
 }
+
+ORDER BY ASC (?lexemeID)
```

Comment:
Can we remove the extra spacing, @Khushalsarode? And also this ORDER BY is not necessary here as we don't need the results in a given order. This will end up just slowing the query down a bit :)

Reply:
sure!

Reply:
done as per requested! 👍

Reply:
shall I mark "resolve conversation"?

Reply:
Can I record this PR to outreachy!